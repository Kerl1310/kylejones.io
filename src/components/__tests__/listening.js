import React from 'react';
import renderer from 'react-test-renderer';

import Listening from '../now/listening';

const testArtist = {
    external_urls: {
        spotify: '6XyY86QOPPrYVGvF9ch6wz'
    },
    name: 'Linkin Park'
}

describe('Listening', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Listening artist={testArtist}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
