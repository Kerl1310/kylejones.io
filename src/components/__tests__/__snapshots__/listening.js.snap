// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Listening renders correctly 1`] = `
<li>
  <strong>
    Listening to: 
  </strong>
  <a
    className="artist-link"
    href="6XyY86QOPPrYVGvF9ch6wz"
    rel="noopener noreferrer"
    target="_blank"
  >
    Linkin Park
  </a>
</li>
`;
